### ISSUE TITLE
Docker footer show the commit instead of tag version in 1.5.0

### ISSUE THREAD

**Issue by EnricoFerro** [go-gitea/gitea#4782]
- Gitea version (or commit ref): 1.5
- Git version:
- Operating system: Docker
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description
In Docker in the footer of the page the displayed version is the commit instead of the version of the tag.
I see this description for the version 1.5.0 / 1.5.0-rc2 / 1.5.0-rc1 / 1.5.0-dev.

The version 1.4.3 show the correct version ( 1.4.3 )
...

## Screenshots

Version 1.5.0:
![image](https://user-images.githubusercontent.com/25525316/44575752-091e2d00-a78d-11e8-8a29-e83e128ca7ad.png)

Version 1.4.3:
![image](https://user-images.githubusercontent.com/25525316/44575982-bd1fb800-a78d-11e8-8f32-d513737eb98d.png)

**Comment by lafriks** [comment]
Are you using tag `1.5` or `1.5.0`?

**Comment by EnricoFerro** [comment]
I use this docker-compose.yml:

```
version: "2"

networks:
        gitea:
            external: false
services:
    gitea:
       container_name: gitea
       image: gitea/gitea:1.5.0
       environment:
           - USER_UID=112
           - USER_GID=117
       restart: always
       networks:
           - gitea
       volumes:
           - /home/git/gitea:/data
       ports:
           - "127.0.0.1:3000:3000"
           - "127.0.0.1:10022:22"
```

and launch:
```
sudo docker-compose up -d --build
```

**Comment by lafriks** [comment]
Hmm, strange. Must be something to do with that standard tag drone feature

**Comment by namadori** [comment]
It seems the fix doesn't work. On 1.6.0 the footer says "0da8bc9".

**Comment by jojomi** [comment]
Still only the hash in 1.7.0 (`3fa49f3`). Would be nice to see this fixed, so successful updates can be verified more easily :).

**Comment by TBK** [comment]
Still an issue with 1.8.0.

**Comment by lunny** [comment]
Do you get gitea binary from the https://dl.gitea.io ?

**Comment by webjoel** [comment]
@lunny Problem is only in Docker..

**Comment by TBK** [comment]
As @webjoel said it is a problem with the Docker build.

**Comment by segevfiner** [comment]
I think `tags: true` doesn't actually exist in Drone, and you need to do `git fetch --tags` by yourself as a separate step to get the tags. See [Drone - --tags](https://docs.drone.io/user-guide/pipeline/cloning/#tags)

**Comment by TBK** [comment]
You can use DRONE_TAG to get the git tag in some circumstances - https://docs.drone.io/reference/environ/drone-tag/

**Comment by sapk** [comment]
The problem is that the variable DRONE_TAG and DRONE_BRANCH that are used in the Makefile are not forwarded to the docker context and so the the build inside docker doesn't take them into account.

**Comment by zeripath** [comment]
Fixed by #6836 and its backport #6839

**Comment by zeripath** [comment]
Hmm actually this one might not be fixed - it depends on when we make the tag. If we tag after building then that's the problem...
